rocky/assets/js/script.js

```diff
@@ -0,0 +1,29 @@
+import { onDomReady } from "../js/imports/utils.js";
```

Comment:
I would rename this file, since it's not bundled with the modal/dialog component in this PR the "script" name is a bit weird and confusing, since it's not very specific. I would suggest naming it dialog.js since that's the HTML element it's targeting.